README.md

```diff
@@ -19,7 +19,21 @@ After downloading the module in the modules folder, visit the Extend (/admin/mod
 
 ## Configuration
 
-@ToDo
+1) Enable the module
+2) visit /media/add to add media content to website.
+3) Visit /admin/structure/media to edit the default media bundles.
+
+### Using Slick to modify gallery display
+
+You can use the slick library to improve the gallery display.
+
+1) Download [Slick Library](https://github.com/kenwheeler/slick/) and extract it in libraries directory. Required assets should be available at /libraries/slick/slick/slick.css
```

Comment:
I would provide a minimal working configuration with as little dependencies as possible (probably just the slick and slick_media modules) and export that.